internal/pkg/image_repositories/build_workflow.go

```diff
@@ -156,7 +156,7 @@ func NewBuildWorkflow(project string) *GithubWorkflow {
 					},
 					{
 						Name: "Setup environment variables",
-						Run:  "echo PROJECT_REF=$(cat manifest.yml | yq '.\"${{ matrix.release }}\".sha') >> $GITHUB_ENV",
+						Run:  "echo PROJECT_REF=$(cat manifest.yml | yq '.\"${{ matrix.release }}\".sha') >> $GITHUB_ENV\necho BUILD_PROJECT_REF=$(git rev-parse --short HEAD) >> $GITHUB_ENV",
```

Comment:
I know in direct command `\n` here will just make things echo to file `$GITHUB_ENVecho`, maybe gitaction perform differently?
If not, we can sue && instead. :)

Reply:
I think it will output like
```yaml
run: |
  echo PROJECT_REF=$(cat manifest.yml | yq '.\"${{ matrix.release }}\".sha') >> $GITHUB_ENV
  echo BUILD_PROJECT_REF=$(git rev-parse --short HEAD) >> $GITHUB_ENV
```